packages/graphviz.rb

```diff
@@ -3,59 +3,53 @@
   homepage 'https://www.graphviz.org/'
-  version '7.1.0'
+  version '9.0.0'
   license 'BSD'
-  compatibility 'all'
+  compatibility 'x86_64 aarch64 armv7l'
   source_url 'https://gitlab.com/graphviz/graphviz.git'
   git_hashtag version
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/graphviz/7.1.0_armv7l/graphviz-7.1.0-chromeos-armv7l.tar.zst',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/graphviz/7.1.0_armv7l/graphviz-7.1.0-chromeos-armv7l.tar.zst',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/graphviz/7.1.0_i686/graphviz-7.1.0-chromeos-i686.tar.zst',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/graphviz/7.1.0_x86_64/graphviz-7.1.0-chromeos-x86_64.tar.zst'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/graphviz/9.0.0_armv7l/graphviz-9.0.0-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/graphviz/9.0.0_armv7l/graphviz-9.0.0-chromeos-armv7l.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/graphviz/9.0.0_x86_64/graphviz-9.0.0-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '7c8572e45f268f52eea6213b5191687f9289ccdfb5bcae98896435f44e91918a',
-     armv7l: '7c8572e45f268f52eea6213b5191687f9289ccdfb5bcae98896435f44e91918a',
-       i686: 'da761f75ea9ccfd0372f6c94b1a87172bb8e89cc8ab257f352cb08117e14fb09',
-     x86_64: 'a695af69c648392c30af266e8916dee9a4940ec650fd9cca70dc07603f8e1018'
+    aarch64: 'c12deae6fa8b9d66b083bcd159658b5f693bd93814f1d6d33736c5944336bd2e',
+     armv7l: 'c12deae6fa8b9d66b083bcd159658b5f693bd93814f1d6d33736c5944336bd2e',
+     x86_64: '87162917d3aeab886e233df4863b76ce830f9d4f5c4ea0f3299210a0be2f40c1'
   })
 
   depends_on 'at_spi2_core' # R
-  depends_on 'cairo' => :build
+  depends_on 'cairo' # R
   depends_on 'expat' # R
   depends_on 'gcc_lib' # R
   depends_on 'gdk_pixbuf' # R
-  depends_on 'glib' => :build
+  depends_on 'ghostscript' # R
   depends_on 'glibc' # R
   depends_on 'glib' # R
-  depends_on 'gtk2' # R
   depends_on 'harfbuzz' # R
   depends_on 'libgd' # R
   depends_on 'libpng' => :build
+  depends_on 'librsvg' # R
   depends_on 'libtool' # R
   depends_on 'libwebp' # R
   depends_on 'libx11' # R
   depends_on 'libxrender' # R
   depends_on 'pango' # R
-  depends_on 'qt5_base' unless ARCH == 'i686'
+  depends_on 'qt5_base' # R
   depends_on 'zlibpkg' # R
-  depends_on 'ghostscript' # R
 
   def self.patch
     system "sed -i 's|/usr|#{CREW_PREFIX}|g' cmake/config_checks.cmake"
-    # system 'patch -Np1 -i graphviz_cmake.patch'
   end
 
   def self.build
-    # gvedit pulls in the QT dependency.
-    @gvedit = ARCH == 'i686' ? 'OFF' : 'ON'
     system "AWK=#{CREW_PREFIX}/bin/mawk \
         cmake -B builddir \
         -G Ninja \
         #{CREW_CMAKE_OPTIONS} \
-        -Dwith_gvedit=#{@gvedit}"
+        -Dwith_gvedit=ON"
```

Comment:
`with_gvedit` is `ON` by default in `9.0.0`

https://gitlab.com/graphviz/graphviz/-/blob/5733d3a95898f1380424ab15f966ace9a283d506/CMakeLists.txt#L10